apps/expo/package.json

```diff
@@ -2,10 +2,10 @@
   "name": "@acme/expo",
   "version": "0.1.0",
   "private": true,
-  "main": "expo-router/entry",
```

Comment:
this should work according to their docs?

Reply:
This should work fine. We did discover an issue in `expo/AppEntry` where we try to import `../../App`, which doesn't work for `.pnpm/expo@.../node_modules/expo/AppEntry.js`.

We know about that issue, and will fix it as soon as we can!